src/main/java/com/rakovets/course/java/core/practice/oop_classes_and_objects/Time.java

```diff
@@ -0,0 +1,46 @@
+package com.rakovets.course.java.core.practice.oop_classes_and_objects;
+
+public class Time {
+    private int hours;
+    private int minutes;
+    private int seconds;
+
+    public Time (int hours, int minutes, int seconds){
+        this.hours = hours;
+        this.minutes = minutes;
+        this.seconds = seconds;
+    }
+    public Time (int totalSeconds){
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"